core/dbt/events/functions.py

```diff
@@ -180,7 +180,7 @@ def create_debug_text_log_line(e: T_Event) -> str:
     if type(e) == MainReportVersion:
         separator = 30 * "="
         log_line = f"\n\n{separator} {get_ts()} | {get_invocation_id()} {separator}\n"
-    color_tag: str = "" if this.format_color else Style.RESET_ALL
+    color_tag: str = "" if not this.format_color else Style.RESET_ALL
```

Comment:
We need the same change in the 'create_info_text_log_line' function at about line 170.